samples/load_table_dataframe_array_contains.py

```diff
@@ -0,0 +1,40 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+def load_table_dataframe_array_contains(table_id):
+
+    # [START bigquery_load_table_dataframe_array_contains]
+
+    from google.cloud import bigquery
+    import pandas
+
+    # Construct a BigQuery client object.
+    client = bigquery.Client()
+
+    # TODO(developer): Set table_id to the ID of the table to create.
+    # table_id = "your-project.your_dataset.your_table_name"
+
+    dataframe = pandas.DataFrame({"A": [[1, 2, 3], [4, 5, 6], [7, 8, 9]]})
+    job = client.load_table_from_dataframe(dataframe, table_id)  # Make an API request.
```

Comment:
Without an explicit schema, this sample is no different from the generic load_table_from_dataframe sample.

I was imagining system test XOR sample, as they are testing the same behavior.